pkg/daemon/ceph/client/mirror_health.go

```diff
@@ -168,6 +168,22 @@ func updateRadosNamespaceStatusMirroring(c *mirrorChecker, mirrorStatus *cephv1.
 		radosNamespace.Status = &cephv1.CephBlockPoolRadosNamespaceStatus{}
 	}
 
+	blockPool := &cephv1.CephBlockPool{}
+	namespaceName := types.NamespacedName{Name: radosNamespace.Spec.BlockPoolName, Namespace: radosNamespace.Namespace}
+	if err := c.client.Get(c.clusterInfo.Context, namespaceName, blockPool); err != nil {
+		if kerrors.IsNotFound(err) {
+			logger.Debug("CephBlockPool %v resource not found for updating the mirroring status, ignoring.", blockPool)
+			return
+		}
+		logger.Warningf("failed to retrieve ceph block pool %q to update mirroring status. %v", namespaceName, err)
+		return
+	}
+
+	if blockPool.Spec.StatusCheck.Mirror.Disabled {
+		logger.Debugf("mirroring status check is disabled for %q", c.namespacedName.Name)
```

Comment:
```suggestion
		logger.Debugf("mirroring status check is disabled for %q", namespaceName)
```
I think we are referring to the blockpoolName here.

Reply:
We are checking blockpool spec and disabling the rados-namespace monitoring

So we are disabling for rados namespace which would be namespaceName